lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -677,6 +684,17 @@ deleteCheckpoints wid = do
     deleteWhere [CheckpointWalletId ==. wid]
     deleteState @s wid -- clear state
 
+-- | Delete the checkpoint associated with a wallet and slotId.
```

Comment:
Better mention that this one is introduced to have guarantee we can put the checkpoints with the same id more than 1 time. Mainly so that the tests can do that and not crash. Otherwise people might wonder why it's needed.

Reply:
done